packages/chakra-ui-docs/docs/tooltip.mdx

```diff
@@ -1 +1,118 @@
+
+<LiveEditor>
+  {`
+  <c-tooltip label="Chakra UI rocks">Psst! Hover Me</c-tooltip>
+`}
+</LiveEditor>
+
+### With an icon
+
+<LiveEditor>
+  {`
+<c-tooltip label="Welcome home" placement="bottom">
+    <c-icon name="phone" />
+</c-tooltip>
+`}
+</LiveEditor>
+
+### With arrow
+
+<LiveEditor>
+  {`
+<c-tooltip has-arrow label="Add new component" placement="right">
+    <c-icon name="add" />
+</c-tooltip>
+`}
+</LiveEditor>
+
+### Tooltip with focusable content
+
+If the children of CTooltip is a focusable element, the tooltip will show when you focus or hover on the element, and will hide when you blur or moves cursor out of the element.
+
+<LiveEditor>
+  {`
+<c-tooltip has-arrow label="Sends your order to Delivery hero" placement="top" bg="blue.600">
+   <c-button>Place Order</c-button>
+</c-tooltip>
+`}
+</LiveEditor>
+
+### Placement
+
+Using the `placement` prop you can adjust where your tooltip will be displayed.
+
+<LiveEditor>
+  {`
+<c-stack should-wrap-children>
+    <c-tooltip has-arrow label="left" placement="left">
+        <c-button variant="outline">Left</c-button>
+    </c-tooltip>
+     <c-tooltip has-arrow label="top" placement="top">
+        <c-button variant="outline">Top</c-button>
+    </c-tooltip>
+     <c-tooltip has-arrow label="right" placement="right">
+        <c-button variant="outline">Right</c-button>
+    </c-tooltip>
+     <c-tooltip has-arrow label="bottom" placement="bottom">
+        <c-button variant="outline">Bottom</c-button>
+    </c-tooltip>
+</c-stack>
+`}
+</LiveEditor>
+
+### Props
+
+| Name                 | Type                 | Default  | Description                                                                           |
+| -------------------- | -------------------- | -------- | ------------------------------------------------------------------------------------- |
+| `isOpen`             | `boolean`            |          | If `true`, the tooltip is shown.                                                      |
+| `defaultIsOpen`      | `boolean`            |          | If `true`, the tooltip is initially shown.                                            |
+| `label`              | `string`             |          | The label of the tooltip.                                                             |
+| `aria-label`         | `string`             |          | An alternate label for screen readers.                                                |
+| `placement`          | `PopperJS.Placement` | `bottom` | Position the tooltip relative to the trigger element as well as surrounding elements. |
+| `children`           | `Vue.VNode`          |          | The `VNode` to be used as the trigger of the tooltip.                                 |
+| `hasArrow`           | `boolean`            |          | If `true` display an arrow tip on the tooltip.                                        |
```

Comment:
Let's remove this `children` from the props table.

React uses `children` where Vue uses slots. So we don't need to add a children prop for any of our component docs.

So we can represent this by adding a **Slots** table. See example shown in the badge component docs:
https://github.com/chakra-ui/chakra-ui-vue/blob/dev/docs/packages/chakra-ui-docs/docs/badge.mdx#slots